socorro/processor/config.py

```diff
@@ -76,3 +76,53 @@
         'wrapped_object_class': 'socorro.external.boto.crashstorage.TelemetryBotoS3CrashStorage',
     },
 })
+
+
+companion_process = DotDict({
+    'companion_class': 'socorro.processor.symbol_cache_manager.SymbolLRUCacheManager',
+    'symbol_cache_size': '40G',
+    'verbosity': 0,
+})
+
+
+new_crash_source = DotDict({
+    'crashstorage_class': 'socorro.external.rabbitmq.crashstorage.RabbitMQCrashStorage',
+    'new_crash_source_class': 'socorro.external.rabbitmq.rmq_new_crash_source.RMQNewCrashSource',
+})
+
+
+processor = DotDict({
+    'jit_classifers.JitCrashCategorizeRule.chatty': True,
```

Comment:
I think this should be:

```
'jit_classifiers': {
    'JitCrashCategorizeRule': {
        'chatty': True
    }
}
```

However, @lonnen is ditching all the chatty things about transform rules, so we could just nix this line--either would be fine.

Reply:
may as well